ocis-reva/docs/configuration.md

```diff
@@ -0,0 +1,656 @@
+---
+title: "Configuration"
+date: "2020-09-21T13:14:49+0200"
+weight: 20
+geekdocRepo: https://github.com/owncloud/ocis-reva
+geekdocEditPath: edit/master/docs
+geekdocFilePath: configuration.md
+---
+
+{{< toc >}}
+
+## Configuration
```

Comment:
Codacy found an issue: [[heading-style] Headings should use setext](https://app.codacy.com/gh/owncloud/ocis/pullRequest?prid=6202733)